crates/rpc-new/types/src/types.rs

```diff
@@ -0,0 +1,245 @@
+//! RPC types for the Orchestration Layer.
+
+use serde::{Deserialize, Serialize};
+use strata_acct_types::{AccountId, BitcoinAmount, MsgPayload};
+use strata_identifiers::{Buf32, OLBlockCommitment};
+use strata_ol_chain_types_new::{Slot, TransactionAttachment};
+use strata_snark_acct_types::{MessageEntry, ProofState, UpdateInputData, UpdateStateData};
+
+/// OL chain status with latest, confirmed, and finalized blocks.
+#[derive(Debug, Clone, Serialize, Deserialize)]
+pub struct RpcOLChainStatus {
+    pub latest: OLBlockCommitment,
+    pub confirmed: OLBlockCommitment,
+    pub finalized: OLBlockCommitment,
+}
+
+/// Epoch summary for an account.
+#[derive(Debug, Clone, Serialize, Deserialize)]
+pub struct RpcAccountEpochSummary {
```

Comment:
This is specific to snark accounts tbf.  Might also want to expose the "final next message idx" for convenience.